### PR TITLE
Clarify docs on Galaxy

### PR DESCRIPTION
**Pull request by samccann**
Galaxy now supports full module documentation, so update the README guidelines accordingly

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Every voice is important. If you have something on your mind, create an issue or
 
 ## Included content
 
-<!-- Galaxy will eventually list the module docs within the UI, but until that is ready, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
+<!-- Galaxy now displays full module docs within the UI. If you don't use Galaxy for your collection, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
```
Comment (felixfontein) [inline README.md:94]: There's still the glaring bug in Galaxy that doesn't show any documentation uses docs fragments from dependent collections. Also it shows plugin docs as well (but no role argument spec docs). How about:
```suggestion
<!-- Galaxy now usually displays full module and plugin docs within the UI. If you don't use Galaxy for your collection, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
```

**Comment by samccann** [comment]
good point. thanks!

**Review by felixfontein — APPROVED** [review]

**Review by russoz — APPROVED** [review]
LGTM

**Review by Andersson007 — APPROVED** [review]

**Comment by Andersson007** [comment]
@samccann thanks for the contribution! @felixfontein @russoz thanks for reviewing!

**Pull request merged**